app/models/spree/product.rb

```diff
@@ -79,14 +76,11 @@ class Product < ApplicationRecord
     around_destroy :destruction
     after_save :update_units
 
+    # -- Scopes
     scope :with_properties, ->(*property_ids) {
       left_outer_joins(:product_properties).
-        left_outer_joins(:supplier_properties).
         where(inherits_properties: true).
-        where(producer_properties: { property_id: property_ids }).
```

Comment:
This looks like a lost feature, which could probably be achieved `through` variants. Is it no longer needed?

Reply:
Producer properties are now handled through the variant, see https://github.com/openfoodfoundation/openfoodnetwork/pull/12562/files#diff-833d4905682446957f9fc24c1b8d02bf9e1cb594484989aa9d8c921a84aab69e